### ISSUE TITLE
Duplicate mount point: /opt/sonarqube/conf

### ISSUE THREAD

**Issue by Janfy** [SonarSource/helm-chart-sonarqube#21]
Hi,

The pod **sonarqube-sonarqube-0** didn't start if:
- **prometheusExporter** is **enabled** 
and
- `sonarProperties` is customized

Both of them declare mount point `/opt/sonarqube/conf` that it's incorrect.

To reproduce this bug, uncomment following lines in `values.yaml`:

```
sonarProperties:
  sonar.forceAuthentication: true
```

Install Sonarqube with comman: `helm upgrade --install -f values.yaml -n sonarqube sonarqube ./`

Result:

```
$ k get pod
NAME                     READY   STATUS                 RESTARTS   AGE
sonarqube-postgresql-0   1/1     Running                1          2d2h
sonarqube-sonarqube-0    0/1     CreateContainerError   0          7m3s
```

```
$ k describe pod sonarqube-sonarqube-0
Events:
...
  Warning  Failed     5s (x3 over 20s)  kubelet            Error: Error response from daemon: Duplicate mount point: /opt/sonarqube/conf
```

Workaround is to disable **prometheusExporter** but it's not the solution.

Regards.

**Comment by tobias-trabelsi-sonarsource** [comment]
Hi @Janfy ,

thanks for the report and your PR. i have included your changes in #27 as there are multiple parts to this issue (see the CI run on your PR)

**Comment by Janfy** [comment]
Thank's, i'll test thoses changes asap.
